<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>MainWindow</class>
 <widget class="QMainWindow" name="MainWindow">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>650</width>
    <height>750</height>
   </rect>
  </property>
  <property name="minimumSize">
   <size>
    <width>650</width>
    <height>750</height>
   </size>
  </property>
  <property name="maximumSize">
   <size>
    <width>650</width>
    <height>750</height>
   </size>
  </property>
  <property name="windowTitle">
   <string>Knihovna</string>
  </property>
  <property name="styleSheet">
   <string notr="true">#MainWindow {
background: rgb(204,229,250);
}
#centralWidget {
background: rgb(204,229,250);
}
#label_2{
background: rgb(204,229,250);
}
#pushButtonKnihy{
background: rgb(235,235,235);
border-radius : 20px;
border : 3px solid rgb( 0,76,153);
}
#pushButton{
background: rgb(235,235,235);
border-radius : 20px;
border : 3px solid rgb( 0,76,153);
color: rgb(0,76,153);
}
#pushButtonZaci{
background: rgb(235,235,235);
border-radius : 20px;
border : 3px solid rgb( 0,76,153);
}
#pushButtonUcitel{
background: rgb(235,235,235);
border-radius : 20px;
border : 3px solid rgb( 0,76,153);
}
#pushButton_5{
background: rgb(235,235,235);
border-radius : 20px;
border : 3px solid rgb( 0,76,153);
}
#pushButtonVypujckaUcitel
{
background: rgb(235,235,235);
border-radius : 20px;
border : 3px solid rgb( 0,76,153);
}
#textBrowser{
background: rgb(242,242,242);
border-radius : 10px;
border : 3px solid rgb( 0,76,153);
}

</string>
  </property>
  <widget class="QWidget" name="centralWidget">
   <layout class="QGridLayout" name="gridLayout">
    <item row="11" column="2">
     <widget class="QTextBrowser" name="textBrowser">
      <property name="styleSheet">
       <string notr="true">#textBrowser{
color: rgb(153,0,0);
}</string>
      </property>
     </widget>
    </item>
    <item row="12" column="2">
     <spacer name="verticalSpacer">
      <property name="orientation">
       <enum>Qt::Vertical</enum>
      </property>
      <property name="sizeHint" stdset="0">
       <size>
        <width>20</width>
        <height>40</height>
       </size>
      </property>
     </spacer>
    </item>
    <item row="4" column="9">
     <spacer name="horizontalSpacer_2">
      <property name="orientation">
       <enum>Qt::Horizontal</enum>
      </property>
      <property name="sizeHint" stdset="0">
       <size>
        <width>40</width>
        <height>20</height>
       </size>
      </property>
     </spacer>
    </item>
    <item row="4" column="2">
     <widget class="QPushButton" name="pushButtonZaci">
      <property name="minimumSize">
       <size>
        <width>350</width>
        <height>40</height>
       </size>
      </property>
      <property name="maximumSize">
       <size>
        <width>350</width>
        <height>40</height>
       </size>
      </property>
      <property name="font">
       <font>
        <pointsize>10</pointsize>
        <weight>75</weight>
        <bold>true</bold>
       </font>
      </property>
      <property name="styleSheet">
       <string notr="true">#pushButtonZaci{
color: rgb(0,67,134)
}</string>
      </property>
      <property name="text">
       <string>          Studenti             </string>
      </property>
     </widget>
    </item>
    <item row="0" column="2">
     <widget class="QLabel" name="labelLogo">
      <property name="text">
       <string/>
      </property>
     </widget>
    </item>
    <item row="1" column="2">
     <widget class="QLabel" name="label">
      <property name="minimumSize">
       <size>
        <width>150</width>
        <height>100</height>
       </size>
      </property>
      <property name="maximumSize">
       <size>
        <width>16777215</width>
        <height>100</height>
       </size>
      </property>
      <property name="font">
       <font>
        <family>Rockwell Extra Bold</family>
        <pointsize>37</pointsize>
        <weight>75</weight>
        <italic>false</italic>
        <bold>true</bold>
       </font>
      </property>
      <property name="styleSheet">
       <string notr="true">#label{
color: rgb(0,76,153);
}</string>
      </property>
      <property name="text">
       <string>Knihovna</string>
      </property>
      <property name="alignment">
       <set>Qt::AlignCenter</set>
      </property>
     </widget>
    </item>
    <item row="5" column="2">
     <widget class="QPushButton" name="pushButton_5">
      <property name="minimumSize">
       <size>
        <width>350</width>
        <height>40</height>
       </size>
      </property>
      <property name="maximumSize">
       <size>
        <width>350</width>
        <height>40</height>
       </size>
      </property>
      <property name="font">
       <font>
        <pointsize>10</pointsize>
        <weight>75</weight>
        <bold>true</bold>
       </font>
      </property>
      <property name="styleSheet">
       <string notr="true">#pushButton_5{
color: rgb(0,67,134)
}</string>
      </property>
      <property name="text">
       <string>      Výpůjčka studentů</string>
      </property>
     </widget>
    </item>
    <item row="2" column="2">
     <widget class="QPushButton" name="pushButtonKnihy">
      <property name="minimumSize">
       <size>
        <width>350</width>
        <height>40</height>
       </size>
      </property>
      <property name="maximumSize">
       <size>
        <width>350</width>
        <height>40</height>
       </size>
      </property>
      <property name="font">
       <font>
        <family>MS Shell Dlg 2</family>
        <pointsize>10</pointsize>
        <weight>75</weight>
        <bold>true</bold>
       </font>
      </property>
      <property name="layoutDirection">
       <enum>Qt::LeftToRight</enum>
      </property>
      <property name="styleSheet">
       <string notr="true">#pushButtonKnihy{
color: rgb(0,67,134)
}</string>
      </property>
      <property name="text">
       <string>            Knihy                </string>
      </property>
      <property name="flat">
       <bool>true</bool>
      </property>
     </widget>
    </item>
    <item row="2" column="0">
     <spacer name="horizontalSpacer">
      <property name="orientation">
       <enum>Qt::Horizontal</enum>
      </property>
      <property name="sizeHint" stdset="0">
       <size>
        <width>40</width>
        <height>20</height>
       </size>
      </property>
     </spacer>
    </item>
    <item row="7" column="2">
     <widget class="QPushButton" name="pushButtonUcitel">
      <property name="minimumSize">
       <size>
        <width>350</width>
        <height>40</height>
       </size>
      </property>
      <property name="maximumSize">
       <size>
        <width>350</width>
        <height>40</height>
       </size>
      </property>
      <property name="font">
       <font>
        <pointsize>10</pointsize>
        <weight>75</weight>
        <bold>true</bold>
       </font>
      </property>
      <property name="styleSheet">
       <string notr="true">#pushButtonUcitel{
color: rgb(0,67,134)
}</string>
      </property>
      <property name="text">
       <string>           Učitelé               </string>
      </property>
     </widget>
    </item>
    <item row="10" column="2">
     <widget class="QLabel" name="label_2">
      <property name="minimumSize">
       <size>
        <width>40</width>
        <height>35</height>
       </size>
      </property>
      <property name="maximumSize">
       <size>
        <width>40</width>
        <height>35</height>
       </size>
      </property>
      <property name="text">
       <string/>
      </property>
     </widget>
    </item>
    <item row="8" column="2">
     <widget class="QPushButton" name="pushButtonVypujckaUcitel">
      <property name="minimumSize">
       <size>
        <width>350</width>
        <height>40</height>
       </size>
      </property>
      <property name="maximumSize">
       <size>
        <width>350</width>
        <height>40</height>
       </size>
      </property>
      <property name="font">
       <font>
        <pointsize>10</pointsize>
        <weight>75</weight>
        <bold>true</bold>
       </font>
      </property>
      <property name="styleSheet">
       <string notr="true">#pushButtonVypujckaUcitel{
color: rgb(0,67,134)
}</string>
      </property>
      <property name="text">
       <string>      Výpůjčka vyučující </string>
      </property>
     </widget>
    </item>
   </layout>
  </widget>
 </widget>
 <layoutdefault spacing="6" margin="11"/>
 <resources/>
 <connections/>
</ui>
